src/_locales/en_US/messages.json

```diff
@@ -512,5 +512,37 @@
                 "example": "Facebook Like"
             }
         }
+    },
+    "sync_intro": {
+        "message": "Cloud sync:<ul><li>Requires Firefox/Chrome Sync</li><li>Upload overwrites any existing data in the cloud</li><li>Download combines the lists of sites where your Badger is disabled</li></ul>",
+        "description": "A brief explanation of how syncing works. Shown above the upload/download cloud data buttons under the Manage Data options page tab."
+    },
+    "upload": {
+        "message": "Upload",
+        "description": "Button label for uploading data to the cloud on Manage Data options page tab."
+    },
+    "upload_cloud": {
+        "message": "Upload disabled sites to cloud",
+        "description": "Button title for uploading data to the cloud on Manage Data options page tab."
+    },
+    "download_cloud": {
+        "message": "Download disabled sites from cloud",
+        "description": "Button title for downloading data from the cloud on Manage Data options page tab."
+    },
+    "download_cloud_success": {
+        "message": "Disabled sites list downloaded from cloud and merged into local list.",
+        "description": "Message displayed to the user after successfully merging cloud data."
+    },
+    "download_cloud_failure": {
+        "message": "Disabled sites list could not be downloaded from cloud.  (You will get this message if you have not uploaded any list to the cloud.)",
```

Comment:
Do we want to append the error message? (Would it be localized?  How likely is it to be too technical for the average user?)  We could instead have a standard message saying to check the error console for details.

Reply:
We could append it. It might be localized, but it's fine either way. It's probably too technical but then we do have clear error text above (something like "Failed to download cloud data"), and displaying the exact error will help us figure out the issue if users start reporting these failures to us. On the other hand, `console.error`-printing them works too since we could just ask users to open the options page console. So ... either is fine, but we should do something.

Reply:
I figured that writing to `console.error` was not a bad idea either way, so that's what I have in there right now.